apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -1,5 +1,9 @@
+import { useState } from "react";
+import { type MultiValue, type SingleValue } from "react-select";
```

Comment:
```suggestion
```

Reply:
I need to use these types for select change function

Reply:
Should I omit this somehow, or export types somewhere in `/libs` folder? Or is this suggestion outdated?

Reply:
no, you don't. only `select.tsx` should know about that we user react-select in our application